reflex/experimental/assets.py

```diff
@@ -1,57 +1,90 @@
 """Helper functions for adding assets to the app."""
 
 import inspect
+import os
 from pathlib import Path
 from typing import Optional
 
 from reflex import constants
 
 
-def asset(relative_filename: str, subfolder: Optional[str] = None) -> str:
-    """Add an asset to the app.
+def asset(
+    path: str,
+    shared: bool = False,
```

Comment:
the `shared` argument is a little bit confusing to me... is `shared=False` for referencing files that are already in the app's `assets/` directory?

i suppose this is to align the API to work with source-relative assets as well as assets that are already in the correct place? But setting the default to `False` seems to break how existing callers are already using this function to reference source-relative assets.

Reply:
`shared=True` ist for librarys which want to expose the assets to other python modules/reflex apps. Shared mode expects the files to be relative to the python source file where `rx._x.asset` is called.

`shared=False` is for "normal"/"local" assets in reflex. It just returns the serving path and checks if the file really exits.